bin/changelog/github/requests.js

```diff
@@ -71,7 +71,9 @@ const fetchAllPullRequests = async ( version ) =>
 		const fetchResults = async ( before ) => {
 			const query = getQuery( milestoneNumber, before );
 			const results = await authedGraphql( query );
-			if ( results.repository.milestone.pullRequests.totalCount < 100 ) {
```

Comment:
Total count will always return 177 no matter what

Reply:
I like `hasPreviousPage` as the condition anyways.  However I still think `totalCount` would be valid because it would break on the _first_ page if there's under 100 total results for the query for _milestone_ based queries. No need to change though because `hasPreviousPage` does the same job.

Reply:
the problem is that `totalCount` will never break the if, causing an infinite loop if we have more than 100.

Reply:
Oh ya you're right, I was under the assumption that the cursor would be empty if there were no previous pages, but that's not the case, using the cursor will just result in the next query returning no results. It's definitely better to use `hasPreviousPage` here to avoid an unnecessary additional request 👍

Reply:
yeah this bugged me this morning, also having cursor as null is like not having it at all, so that’s page 1 again, page 2, page empty, page 1, page 2, page empty, page.....
I left my laptop for like 15 minute waiting for something to happen 😅